### PR TITLE
DAOS-16669 test: fix pool list ftest

### PR DESCRIPTION
**Pull request by knard-intel**
### Description

Fix regression on pool/list_verbose.py functional test introduced with DAOS-14419.

### Before requesting gatekeeper:

* [x] Two review approvals and any prior change requests have been resolved.
* [x] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [x] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [x] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [x] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate owners.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

### CODE REVIEW

File: utils/ansible/ftest/vars/Rocky8.yml [hunk 1]
```diff
@@ -32,9 +32,11 @@ daos_base_deps:
 daos_dev_deps:
   rpms:
     - daos-raft-devel
+    - daxctl-devel
```
Comment (tanabarr) [inline utils/ansible/ftest/vars/Rocky8.yml:35]: nit: unrelated changes should probably be in their own PR to separate concerns

**Review by tanabarr — DISMISSED** [review]

File: utils/ansible/ftest/vars/Rocky8.yml (hunk 1 above)
Comment (daltonbohning) [reply utils/ansible/ftest/vars/Rocky8.yml:35]: I agree this should be separate. And since it's just ansible - not used in CI - we don't need to run all the Jenkins validation for a separate PR that just updates ansible

**Review by daltonbohning — COMMENTED** [review]
ftest LGTM but I agree with Tom on moving the ansible changes to a separate PR

File: utils/ansible/ftest/vars/Rocky8.yml (hunk 1 above)
Comment (knard-intel) [reply utils/ansible/ftest/vars/Rocky8.yml:35]: - [ ] Remove the ansible script commit

**Review by shimizukko — DISMISSED** [review]

File: utils/ansible/ftest/vars/Rocky8.yml (hunk 1 above)
Comment (knard-intel) [reply utils/ansible/ftest/vars/Rocky8.yml]: > * [x]  Remove the ansible script commit

Fixed with commit 6b1c1504f

**Review by shimizukko — APPROVED** [review]

**Review by phender — APPROVED** [review]

**Review by daltonbohning — APPROVED** [review]

**Review by tanabarr — APPROVED** [review]
